scripts/deblur

```diff
@@ -77,12 +87,33 @@ def deblur_seqs(seqs_fp, read_error, mean_error, error_dist, indel_prob,
     logger = logging.getLogger(__name__)
     logger.info('deblur deblur_seqs started on file %s' % seqs_fp)
 
+    # set the error distribution
+    if error_dist is None:
+        if read_error is None:
+            logger.info('using default error profile')
+            # set to the default illumina read error profile
+            error_dist = default_error_dist
+        else:
+            # If the user provided an error_dist value,
+            # we map it to a list of floats
+            logger.info('creating error_dist from read_error')
+            error_dist = create_error_profile(read_error)
+    else:
+        if read_error is not None:
+            # if we get both read_error and error_dist it's an error
+            # since they both set the same variable
+            logger.error('Cannot use both --error_dist and --read_error')
+            raise ValueError("Cannot use both --error_dist and --read_error")
+
+    logger.info('error_dist is : %s' % error_dist)
+    error_dist = list(map(float, error_dist.split(',')))
```

Comment:
This call will raise an exception when `error_dist is None` and `read_error is not None` as the return type from `create_error_profile` is a `list` of `float` and not a `str`.